docs/zk-stack/running-a-hyperchain/locally.md

```diff
@@ -27,20 +33,130 @@ zk
+
+Once you have your <a href="https://github.com/matter-labs/zksync-era" target="_blank">zksync-era</a> repo set up locally, you can run the following command to build environment and configuration files for block explorer based on your <a href="https://github.com/matter-labs/zksync-er" target="_blank">zksync-era</a> repo configuration:
+
+```bash
+zk server --components "http_api,eth,state_keeper,housekeeper"
+```
+
+The script generates all the necessary configuration files for block-explorer, which you can edit if you need any changes.
+
+## Enabling Boojum prover
+
+With the default configuration, your hyperchain is not running a prover, and has a DummyExecutor contract, which mainly “accepts” that a batch is executed without proof. This enables you to test it with much lower hardware requirements.
+
+To enable the prover, run the `zk stack prover-setup` command. It will guide through the necessary configuration.
+
+:::warning
+
+<u>Running a prover is not required</u> for deploying a testnet. The requirements below are only necessary if you want to enable the prover.
+:::
+
+### Requirements for <u>GPU</u> Prover
+
+The docker compose file assumes you will be running all components in the same machine. The current minimum requirements for a low TPS scenario are:
+
+- 16 GB VRAM NVIDIA GPU
+- 16 Core CPU
+- 64 GB of RAM
+- 300 GB of Disk Space (SSD preferred)
+
+### Requirements for <u>CPU</u> Prover
+
+The docker compose file assumes you will be running all components in the same machine. The current minimum requirements for a low TPS scenario are:
+
+- 32 Core CPU
+- 128 GB of RAM
+- 700 of Disk Space (SSD preferred)
+
+## Addendum
+
+- If you make changes to any contract, you can always deploy a new hyperchain to easily test those changes.
+
+- If you configure your hyperchain once, you don't need to do it again as the wizard allows you to use an existing config file.
+
+- For now, it is only possible to deploy a hyperchain as an L2, but soon it will also work as L3s.
+
+- When running the default matterlabs/geth, you have a set of rich wallets available to you. You can check them <a href="https://github.com/matter-labs/local-setup/blob/main/rich-wallets.json" target="_blank">here</a>.
+
+- If you want to have a custom local base chain, you must ensure you have a database for your hyperchain, as well as the local RPC for your L1.
+
+  - To run a Postgres 14 database for your hyperchain, execute the following:
+
+```bash
+docker-compose -f docker-compose-zkstack-common.yml up -d postgres
+```
+
+in case you don't want to use the docker Postgres database above but another one you already have locally, make sure its version is 14 and it is running and accepts connections at `postgres://postgres@localhost/zksync_local`. You can test with:
```

Comment:
nit: Uppercase start of sentence
```suggestion
In case you don't want to use the docker Postgres database above but another one you already have locally, make sure its version is 14 and it is running and accepts connections at `postgres://postgres@localhost/zksync_local`. You can test with:
```